en/03_Formalicious/v2/01_Installation.md

```diff
@@ -0,0 +1,29 @@
+```
+
+Warning updating Formalicious V2 requires some changes in templating. 
+
+Changes have been made in naming of snippets and placeholders. 
+
+Please read the instructions before updating to Formalicious V2.
+```
+
+Installing Formalicious requires the free Extra _FormIt 4.1.1-pl_ to function properly. FormIt is also maintained by us and thus, should guarantee maximum compatibility between FormIt and Formalicious. The MODX installer should install this automatically when installing Formalicious from the modmore repository.
+
+A custom manager page called _Formalicious_ is installed within the _Extras_ menu. Also a Template Variable and Element Category named _Formalicious_ are created. The _Template Variable_ should be assigned to all the templates you wish to use Formalicious forms on and a Snippet should be added to those templates.
+Access to Formalicious and the admin panel can be set at the modx ACL access policies. 
+
+To accomplish this, follow these steps:
+1. Click the _formalicious_ TV
+2. Click _Template Access_
+3. Enable the checkmarks for the templates on which you want to use Formalicious-forms
+4. Save your changes
+5. Go to every template you just enabled
+6. In the field _Template code (HTML)_, add the following Snippet-call at the location where you want to show your forms: ```[[!FormaliciousRenderForm? &form=`[[!*formalicious]]`]]``` We recommend to add it just below the ```[[*content]]``` placeholder, because it enables your content-managers to add text above the form. 
```

Comment:
```suggestion
6. In the field _Template code (HTML)_, add the following Snippet-call at the location where you want to show your forms: ```[[!FormaliciousRenderForm? &form=`[[*formalicious]]`]]``` We recommend to add it just below the ```[[*content]]``` placeholder, because it enables your content-managers to add text above the form. 
```